packages/cyberstorm/src/components/componentStyles/PackageCard.module.css

```diff
@@ -0,0 +1,140 @@
+/* Common PackageCard styles */
+.root {
+  display: flex;
+  flex: none;
+  flex-direction: column;
+  gap: var(--gap-m);
+  align-items: flex-start;
+
+  border-radius: var(--border-radius-m);
+  overflow: hidden;
+}
+
+.root:hover {
+  box-shadow: var(--box-shadow-1);
+}
+
+.imageWrapper {
+  position: relative;
+  width: 100%;
+  padding-bottom: 100%;
+  overflow: hidden;
+  transition: 0.2s;
+}
+
+.root:hover .imageWrapper {
+  filter: brightness(1.15);
+}
+
+/* Styles for PackageCard's image */
+.image {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 100%;
+  height: auto;
+  transform: var(--transform-translate);
+  transition: 0.2s;
+
+  --transform-translate: translate(-50%, -50%);
+}
+
+.root:hover .image {
+  transform: var(--transform-translate) scale(1.05);
+}
+
+.flagWrapper {
+  display: flex;
+  flex-direction: column;
+  gap: var(--gap-s);
+}
+
+.content {
+  display: flex;
+  flex: none;
```

Comment:
Is this intentional since two rows below you also set `flex-grow: 1`? You could leave only that one in, or if you're sure you need to set `flex-shrink` to 0, you can shorthand everything with `flex: 1 0 auto`.

Reply:
Removed :D